layer/nrlf/core/tests/test_validators.py

```diff
@@ -599,6 +599,195 @@ def test_validate_category_coding_invalid_system():
     }
 
 
+def test_validate_content_extension_too_many_extensions():
+    validator = DocumentReferenceValidator()
+    document_ref_data = load_document_reference_json("Y05868-736253002-Valid")
+
+    document_ref_data["content"][0]["extension"].append(
+        {
+            "url": "https://fhir.nhs.uk/England/StructureDefinition/Extension-England-ContentStability",
+            "valueCodeableConcept": {
+                "coding": [
+                    {
+                        "system": "https://fhir.nhs.uk/England/CodeSystem/England-NRLContentStability",
+                        "code": "static",
+                        "display": "static",
+                    }
+                ]
+            },
+        }
+    )
+
+    result = validator.validate(document_ref_data)
+
+    assert result.is_valid is False
+    assert result.resource.id == "Y05868-99999-99999-999999"
+    assert len(result.issues) == 1
+    assert result.issues[0].dict(exclude_none=True) == {
+        "severity": "error",
+        "code": "invalid",
+        "details": {
+            "coding": [
+                {
+                    "system": "https://fhir.nhs.uk/ValueSet/Spine-ErrorOrWarningCode-1",
+                    "code": "INVALID_RESOURCE",
+                    "display": "Invalid validation of resource",
+                }
+            ]
+        },
+        "diagnostics": "Invalid content extension length: 2 Extension must only contain a single value",
+        "expression": ["content[0].extension"],
+    }
+
+
+def test_validate_content_extension_invalid_code():
```

Comment:
What about when only one of them is invalid?

Reply:
we dont check the display so it wont matter as long as the code is correct, but I can add in that check because code and display should be the same value anyways static or dynamic and add that test case

Reply:
Hmm... maybe we should be checking for the display then? Probably in another ticket. But is there any reason why the display would be different than the expected value?

Reply:
added the new case and validation